SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -26,6 +24,7 @@ BuildRequires:  pkgconfig
 BuildRequires:  systemd-devel
 BuildRequires:  systemd-rpm-macros
 BuildRequires:  zlib-devel
+BuildRequires:  luajit-devel
```

Comment:
Since the list was already in alphabetical order, could you move the new line up a bit?